### DISCUSSION TITLE
Proposal for `neanes` GitHub organization

### DISCUSSION THREAD

**Discussion by basil** [neanes/neanes#487] [General]
First and foremost, I would like to express my deep gratitude for the invaluable contribution of Neanes to the open source software community. We are truly fortunate to have such a remarkable piece of open source software.

In _Producing Open Source Software_, Karl Fogel [writes](https://producingoss.com/en/social-infrastructure.html):

> This chapter tries to show the structural properties successful projects have in common. I mean “successful” not just in terms of technical quality, but in terms of operational health and survivability. Operational health is the project’s ongoing ability to incorporate new code contributions and new developers […]. Survivability is the project’s ability to continue independently of any individual participant or sponsor — think of it as the likelihood that the project would continue even if all of its founding members were to move on to other things. This is also known as the “Bus Factor”, that is, how many participants would have to get hit by a bus (figuratively speaking) for the project to be unable to continue.

In the spirit of ensuring the long-term operational health and survivability of the project, I would like to propose the following steps:

1. **Creation of a `neanes` GitHub organization:** To reflect the project’s growth and foster a stronger sense of community identity, I propose establishing a dedicated `neanes` GitHub organization. This move also mitigates potential risks associated with a personal GitHub account, given life’s unpredictable nature.
2. **Transfer of the Neanes family of repositories:** As part of this transition, I propose transferring the Neanes family of Git repositories to the `neanes` GitHub organization. This will provide a centralized hub for collaboration and management.
3. **Addition of myself as co-maintainer:** Recognizing the inevitability that open source projects eventually outgrow their founder’s direct day-to-day involvement, I would like to offer my assistance as a co-maintainer. This step aims to enhance the project’s overall survivability and provide a support structure for times when you may need to step back, even temporarily. I want to emphasize that my role as co-maintainer is not intended to alter existing project dynamics; rather, it is to complement and support the project’s continued success.

To facilitate these changes, I have [taken the initiative](https://github.com/neanes) to register the `neanes` GitHub organization. If you would like to proceed with this proposal, I would be honored to invite you as another GitHub organization administrator, allowing for a seamless transfer of repositories.

I understand the significance of this proposal and assure you of my respect for your vision and decision-making authority. Your leadership in a [BD](https://producingoss.com/en/benevolent-dictator.html) role has been essential in getting the project to this point, and it will remain essential as we work toward a 1.0 release. I look forward to your consideration of this proposal and thank you once again for your dedication.

**Comment by danielgarthur** [comment]
I accept the proposal. Please invite me to the organization.

**Reply by basil** [reply]
Great! I just sent you an invitation to become an owner of the GitHub organization.

**Reply by basil** [reply]
Looks like you successfully accepted the invitation. As an org admin, you should now be able to transfer the repositories as described [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository).

**Reply by basil** [reply]
A related question is the web site URL. I just registered `neanes.org`. If desired, I can keep it renewed and managed as part of my portfolio of domains, add you as a contact, or even transfer the domain to you if you want it. I think it would work well to have the main web site at `neanes.org` and the webapp at `webapp.neanes.org` and I can set this up tomorrow if desired.

**Reply by basil** [reply]
Should we move the `neanes-web-app` and `sbmufl` repositories next? I'm happy to help reconfigure things once they're in their new home.
